### ISSUE TITLE
Failed to install

### ISSUE THREAD

**Issue by mupsje** [Skons/hassio-addons#45]
Hi.

I get error when I install in new Hassio.
version : 2023.4.16.1

HASSIO
Home Assistant 2023.4.5
Supervisor 2023.04.0
Frontend-versie: 20230411.1 - latest

```
The command '/bin/bash -o pipefail -c apk add --no-cache --virtual .build-dependencies tesseract-ocr tesseract-ocr-dev jpeg-dev libjpeg zlib-dev gcc libc-dev' returned a non-zero code: 85
```

**Comment by Skons** [comment]
The error comes down to this
```
ERROR: unable to select packages:

  musl-1.2.2-r1:
    breaks: musl-dev-1.2.2-r2[musl=1.2.2-r2]
```

When removing libc-dev from the installation, there is no problem. Probably updating the python base image solves the problem. Currently testing.

**Comment by mupsje** [comment]
thanks, is this something we can do ourself?

**Comment by Skons** [comment]
Unless you start developing this addon, no. I had to update the base image and with that update the way the DGP stuff interacted with docker. That's all fixed for now with the release of 2023.4.21.1.

**Comment by mupsje** [comment]
Thanks allot
